### PR TITLE
#5 - Replace npm with yarn in husky hook

### LINKED ISSUE #5: Husky hooks, replace npm with yarn
Husky hooks, replace npm with yarn:

"pre-commit": "yarn lint:fix && yarn test",
 "pre-push": "yarn test"

### PR DESCRIPTION
**Pull request by krajasekhar-unify**
Fix #5

### CODE REVIEW

**Review by nirajkvinit — APPROVED** [review]
It looks good. A gentle reminder, you should follow the guidelines we have prepared

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,5 @@ dist
 
 # TernJS port file
 .tern-port
+
+yarn.lock
```
Comment (sudheer-sangaraju) [inline .gitignore]: @nirajunify you've mentioned we've to keep package-lock.json under version control, Does same is applied for yarn.lock or not?

**Comment by sudheer-sangaraju** [comment]
@krajasekhar-unify what is the strategy that we're gonna follow branch name? I see you have separated by _

File: .gitignore (hunk 1 above)
Comment (nirajunify) [reply .gitignore]: Yes. Yarn.lock should be under version control. @krajasekhar-unify please update the code

**Review by nirajunify — COMMENTED** [review]
> @krajasekhar-unify what is the strategy that we're gonna follow branch name? I see you have separated by _

Valid branching name strategy is 
<issue-type>/<issue-no>-<basic-issue-description>

**Review by nirajunify — CHANGES_REQUESTED** [review]
Yarn.lock should be under version control

You should remove it from .gitignore

File: .gitignore (hunk 1 above)
Comment (krajasekhar-unify) [reply .gitignore]: Updated

**Comment by krajasekhar-unify** [comment]
> Yarn.lock should be under version control
> 
> You should remove it from .gitignore

Resolved.

**Review by nirajunify — APPROVED** [review]

**Review by zulqer-nain — APPROVED** [review]

**Pull request merged**